core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1091,7 +1091,7 @@ protected function generateGlobalButtons()
 			if (\is_array($v['button_callback'] ?? null))
 			{
 				$this->import($v['button_callback'][0]);
-				$return .= $this->{$v['button_callback'][0]}->{$v['button_callback'][1]}($v['href'] ?? '', $label, $title, $v['class'], $attributes, $this->strTable, $this->root);
+				$return .= $this->{$v['button_callback'][0]}->{$v['button_callback'][1]}($v['href'] ?? null, $label, $title, $v['class'] ?? null, $attributes, $this->strTable, $this->root);
```

Comment:
Why would `href` suddenly return null rather than `''`?  That's a behavior change?

Reply:
Its now the same as in the line below. It is also better imho because it allows the callback to distinguish whether the option was set or not.

Reply:
We also use `?? null` in lines 929, 935 and 1100, so this is the only place where it is `?? ''` and thus the change is correct IMHO.